src/relax/op/image/resize.cc

```diff
@@ -122,7 +122,8 @@ TVM_REGISTER_OP("relax.image.resize2d")
     .add_argument("size", "Shape", "The output image shape.")
     .set_attr<FInferStructInfo>("FInferStructInfo", InferStructInfoResize2D)
     .set_attr<FRelaxInferLayout>("FRelaxInferLayout", InferLayoutResize2d)
-    .set_attr<TMixedPrecisionPolicy>("TMixedPrecisionPolicy", MixedPrecisionPolicyKind::kFollow);
+    .set_attr<TMixedPrecisionPolicy>("TMixedPrecisionPolicy", MixedPrecisionPolicyKind::kFollow)
+    .set_attr<Bool>("FPurity", Bool(true));
```

Comment:
seems like all ops have to be marked as pure except `call_dps_packed` and `call_packed`.
cc @junrushao  if there is any implication of this for op_schema which is WIP.